_data/sitetext.yml

```diff
@@ -0,0 +1,32 @@
+
+# Configuration: Add to _config.yml
+#
+# How to use:
+# {{ site.data.sitetext.<var_name> | default: 'text' }}
+
+header:
+  title: Welcome to Deliberation Lab!
+  text: It's Nice To Meet You
+  button: Tell Me More
+  buttonlink: "#tools"
+
+footer:
+  legal: "Privacy Policy"
+  social:
+    - url: https://twitter.com
+      icon: "fab fa-twitter"
```

Comment:
Where do these icons come from? If I want to add something instead of twitter down the road, (ie, google scholar, etc) where would I find a list of options?